wis2node.env

```diff
@@ -6,6 +6,7 @@ WIS2NODE_DATADIR_PUBLIC=$WIS2NODE_DATADIR/data/public
 
 WIS2NODE_OSCAR_API_TOKEN=some_token
 WIS2NODE_URL=http://localhost/
+WIS2NODE_PIP3_EXTRA_PACKAGES="elasticsearch<8"
```

Comment:
Same here; shouldn't this be for wis2node plugins? If wis2node needs elastic, then this should be in `requirements.txt`.